eslintrc.json

```diff
@@ -27,12 +27,14 @@
 		"keyword-spacing": [2, { "before": true, "after": true }],
 		"linebreak-style": 2,
 		"no-empty": [2, { "allowEmptyCatch": true }],
-		"no-invalid-this": 2,
 		"no-loop-func": 2,
 		"no-script-url": 2,
 		"no-shadow": 2,
 		"no-tabs": 0,
+		"no-template-curly-in-string": 1,
+		"no-throw-literal": 1,
```

Comment:
I think we should encourage returning an `Object`, but if you only need a `String`, why not do it?